src/pages/TicketPage/TicketPage.tsx

```diff
@@ -276,6 +267,19 @@ export const TicketPage = () => {
                         </Grid>
                     </AccordionDetails>
                 </TicketAccordionWrapper>
+                <Button
```

Comment:
Suggestion that go home button may be placed in the same line as download button. The two primary buttons appear distracting.

Reply:
Done